src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Mono.cs

```diff
@@ -0,0 +1,2697 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace System
+{
```

Comment:
I assume that this is copy&paste without any changes.

Reply:
That's my understanding. But I'm trying to make sure I understand the point in history from which it was all copied. @adamsitnik or I will circle back here to edit the description to include notes pointing to the source of the copy.